config/webpack.config.js

```diff
@@ -33,7 +44,79 @@ const globalEntryFiles = {
   ],
 };
 
-const configGenerator = (buildOptions, apps) => {
+/**
+ * Get a list of all the entry points.
+ *
+ * If the `entry` CLI argument is passed, only the specified
+ * application entries are built.
+ */
+function getEntryPoints(entry) {
+  const manifests = getAppManifests();
+  let manifestsToBuild = manifests;
+  if (entry) {
+    const entryNames = entry.split(',').map(name => name.trim());
+    manifestsToBuild = manifests.filter(manifest =>
+      entryNames.includes(manifest.entryName),
+    );
+  }
```

Comment:
:thinking: True. It would produce unexpected results if we have two apps where one name is a subset of another, though. For example, `{ appName: '526' }` and `{ appName: '526-all-claims' }`. We don't right now, but it's a thing to watch out for.

Reply:
Good point.

Reply:
Should we specifically look for that and throw a build error with solid context to prevent a developer accidentally adding a duplicate name?

Reply:
We definitely could provide better error handling in here! Since we don't do that right now, I'm going to leave it off of this PR. The goal here is to provide 1:1 functionality with the old way before we make any improvements. (Trying to make as small of changes as I can get away with.)

Reply:
👍